api/trace_service.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
+
+package envoy.api.v2;
+
+import "api/base.proto";
+import "api/grpc_service.proto";
+import "trace.proto";
+
+import "google/api/annotations.proto";
+
+import "validate/validate.proto";
+
+// Service for streaming traces to server that consumes the trace data. It
+// uses OpenCensus data model as a standard to represent trace information.
```

Comment:
Remove "as a standard", since it doesn't provide much value.

Please add a link to OpenCensus